dkist/dataset/tests/test_dataset.py

```diff
@@ -43,20 +43,23 @@ def test_init_missing_meta_keys(identity_gwcs):
         Dataset(data, wcs=identity_gwcs, meta={"headers": {}})
 
 
+@pytest.mark.accept_cli_datasets
```

Comment:
I suggest we add a kwarg to this mark which tells it *which* argument to replace with the cli argument, so in this case it would be:

```suggestion
@pytest.mark.accept_cli_datasets(fixture="dataset")
```

we could make it default to "first argument" if we want, but this might be more explicit.

Reply:
Good call, I was wondering how to make it more robust in cases where there are multiple fixtures.